### ISSUE TITLE
How to add assets

### ISSUE THREAD

**Issue by snoblenet** [makehuman-js/makehuman-data#1]
Hi Mike,

I want to add clothing to makehuman-data using resources that could be provided in any number of formats. E.g: http://www.makehumancommunity.org/clothes/female_yoga_suit.html

What format are the assets you have saved to this repo and is there a recommended process for converting outside models to this format?

Thanks,

Steven

**Comment by wassname** [comment]
Clothing is modular, I used a [jupyter-notebook](https://github.com/makehuman-js/makehuman-data/blob/master/scripts/notebooks/Export_proxies_to_threejs.ipynb) to convert all proxies (clothing, accessories) to threejs format.

It's a bit messy since I just ran it once, and no one else has. 

- put your clothing/proxies into your makehuman data directory (this changes based on system) so makehuman loads it
- setup makehuman-data as per the readme.md (involved clonins makehuman and downloading assets, then compiling assets). 
- Then run `Export_proxies_to_threejs.ipynb`. It will write your threejs data to a temporary directory.
- Copy them into the `node_modules/makehuman-data/public/data/proxies` directory in your project, or where ever your makehumanjs data is located
- The last thing to do is update `node_modules/makehuman-data/public/data/resources.json` (or wherever your data is) which is an index of available proxies.
